.github/actions/semver/action.yaml

```diff
@@ -0,0 +1,64 @@
+name: 'semver'
+description: 'Provides sevmer output for current git repo'
+inputs:
+  debug:
+    required: false
+    default: false
+  file:
+    required: false
+    default: ''
+  process_dotnet:
+    required: false
+    default: false
+outputs:
+  version:
+    description: Json based string with version values
+    value: ${{ steps.version.outputs.json}}
+runs:
+  using: "composite"
+  steps:
+    - id: set-path
+      shell: bash
+      run: echo "${{ github.action_path }}/bin" >> $GITHUB_PATH
+    - id: debug
+      shell: bash
+      if: ${{ inputs.debug == 'true' }}
+      run: |
+        echo "VERSIONING_DEBUG=true" >> "$GITHUB_ENV"
+    - id: version
+      shell: bash
+      run: |
+        set -o pipefail
+        JSON=$(semver)
+        if [[ -z ${JSON} ]]; then
+          exit -1
+        else
```

Comment:
No need in `else` block here, cause you already did `exit -1` in if. so you can do just like this:

```
set -o pipefail
JSON=$(semver)
if [[ -z ${JSON} ]]; then
  exit -1
fi
echo "json=${JSON}" >> $GITHUB_OUTPUT
```

Reply:
Done